libs/ml/include/ml/utilities/graph_builder.hpp

```diff
@@ -551,6 +552,21 @@ void BuildNodeAndInsertTrainables(NodeSaveableParams<T> const &nsp, std::string
     g->AddTrainable(node, name);
     break;
   }
+  case OpType::METRIC_CATEGORICAL_ACCURACY:
```

Comment:
Since you've implemented a new op you'll need to update the graph test which adds one of every op type as well